README.md

```diff
@@ -362,6 +363,27 @@ MACOSX_DEPLOYMENT_TARGET=10.9 CC=clang CXX=clang++ python setup.py build --cmake
 ccmake build  # or cmake-gui build
 ```
 
+##### Installation Issue
+
+Sometimes you would encounter PyTorch local source is not being detected by Python interpreter after building PyTorch above.
+To resolve this, you may try the following steps:
+
+1. Remove the previously installed package:
+
+    ```bash
+    python setup.py develop --uninstall
+    ```
+
+2. Change directory to the root of the PyTorch project directory.
+
+3. Install the package in editable (develop) mode again by running:
+
+    ```bash
+    python -m pip install -e .
```

Comment:
We actually don't recommend `pip install -e .` because the pytorch build takes a long time and `python setup.py` develop is more verbose